src/app/dao/tests/ConfigRoomDAO.spec.ts

```diff
@@ -254,16 +254,28 @@ describe('ConfigRoomDAO', () => {
             // Then it should fail
             await expectPermissionToBeDenied(result);
         });
-        it('should forbid to add themself to the candidates', async() => {
+        it('should forbid to join the list of candidates', async() => {
             // Given a part with a configRoom, with the current user being the creator
             const createdPart: CreatedPart = await createPart({ createConfigRoom: true });
 
-            // When trying to add themself to the candidates
+            // When trying to join the list of candidates
             const result: Promise<void> = configRoomDAO.addCandidate(createdPart.partId, createdPart.creator);
 
             // Then it should fail
             await expectPermissionToBeDenied(result);
         });
+        it('should allow to remove someone else from the candidates', async() => {
```

Comment:
no further condition, like timeout ?
And a test to say "but not when not timed out" ?

Reply:
nope, tu dois pouvoir supprimer les candidats à la suppression de la configRoom, peu importe leur timeout ou pas

Reply:
Y a pas un moyen de double check que ça permet pas de just kicker tout le monde sans raison :p ? (seul creator peut kicker à tout va, mais seulement avant le début)

Reply:
non !

Reply:
Bah, on en as, par-ci par-là, des test "ifCreatorHasTimedOut", non ?

Reply:
mais c'est créateur qui supprime le candidat ici !